bouncer/util.py

```diff
@@ -65,9 +82,20 @@ def parse_document(document):
         raise InvalidAnnotationError(
             _("The annotation has no URI"), "annotation_has_no_uri")
 
+    if quote is None:
```

Comment:
This whole `if` block is never being called because a few lines before this you check if `quote` is `None` and if so, you give it a boilerplate value. It should be removed.